guides/common/assembly_managing-errata.adoc

```diff
@@ -1,55 +1,4 @@
-[id="Managing_Errata_{context}"]
-= Managing Errata
-
-ifdef::satellite[]
-As a part of Red Hat's quality control and release process, we provide customers with updates for each release of official Red Hat RPMs.
-Red Hat compiles groups of related package into an *erratum* along with an advisory that provides a description of the update.
-endif::[]
-ifndef::satellite[]
-{RHEL} users receive errata for each release of official Red Hat RPMs.
-SUSE Linux Enterprise Server users receive errata for each release of official SUSE RPMs.
-endif::[]
-ifdef::orcharhino[]
-Additionally, ATIX provides Debian and Ubuntu users with errata.
-endif::[]
```

Comment:
Perhaps this should stay in the assembly.
Mark -- see my comment for `assembly_synchronizing-content-between-foreman.adoc`